clang/lib/Sema/SPIRVBuiltins.td

```diff
@@ -911,52 +911,68 @@ foreach name = ["MemoryBarrier"] in {
 // 3.32.21. Group and Subgroup Instructions
 
 foreach name = ["GroupAsyncCopy"] in {
-  // TODO: Allow enum flags instead of UInt ?
-  // TODO: We should enforce that the UInt must be a literal.
-  def : SPVBuiltin<name, [Event, UInt, PointerType<AGenTypeN, LocalAS>, PointerType<ConstType<AGenTypeN>, GlobalAS>, Size, Size, Event], Attr.Convergent>;
-  def : SPVBuiltin<name, [Event, UInt, PointerType<AGenTypeN, GlobalAS>, PointerType<ConstType<AGenTypeN>, LocalAS>, Size, Size, Event], Attr.Convergent>;
+  // TODO: Allow enum flags instead of Int ?
+  // TODO: We should enforce that the Int must be a literal.
+  def : SPVBuiltin<name,
```

Comment:
Same as in https://github.com/intel/llvm/pull/17438#issuecomment-2731623260 , should we have a test checking that clang emits correct signature and mangling for the builtin in the IR?

Reply:
thanks @Fznamznon 
done in https://github.com/intel/llvm/pull/17445/commits/bc42ed15d414eec77f232af783919754a0b69a82